src/thing.cpp

```diff
@@ -9,13 +9,9 @@
 
 const Position& Thing::getPosition() const
 {
-	const Tile* tile = getTile();
+	auto tile = getTile();
```

Comment:
Could this be `const auto`
```suggestion
	const auto tile = getTile();
```